### DISCUSSION TITLE
Don't prompt for app install on public pages

### DISCUSSION THREAD

**Discussion by designatedsuccessor** [outline/outline#7197] [Ideas]
For publicly shared pages, don't prompt the user to download the Outline app (e.g., the Android app).  In our testing, users are confused and think the site may be compromised.  When they go to a public site, they expect it to act like any other public site.
On private pages, the installation prompt makes more sense because the user knows and understands that they may edit the page, so the Outline app is naturally just a tool to do that.

**Comment by tommoor** [comment]
Agreed, created an issue
